server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -219,6 +219,13 @@ public class SearchService extends AbstractLifecycleComponent implements IndexEv
         Property.Dynamic
     );
 
+    public static final Setting<Boolean> QUERY_PHASE_PARALLELISM_ENABLED = Setting.boolSetting(
+        "search.query_phase_parallelism_enabled",
```

Comment:
let's call it `search.query_phase_parallel_collection_enabled`? This is mostly to remind myself as well as others that it won't control parallelism applied to query rewrite.